pom.xml

```diff
@@ -38,6 +38,12 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- https://mvnrepository.com/artifact/jstl/jstl -->
```

Comment:
please formated the dependency using java formatted

Reply:
I did the formatting